### DISCUSSION TITLE
When will Clarity v6 be released?

### DISCUSSION THREAD

**Discussion by wghglory** [vmware/clarity#6710] [Q&A] [answered]
Do we have clarity v6 documentation? When will datagrid be released?

**Comment by bbogdanov** [comment]
@coryrylan

**Comment by mathisscott** [comment, marked as answer]
Thanks for reaching out, @wghglory 

We presently have no timeline for the stable release of `@cds/core` v6.0. This includes the merge and release of the Core datagrid and signposts which are both feature and code complete pending accessibility review.

Our new manager, Steve, sets the priorities for the team so I recommend reaching out to him at strefethen@vmware.com with any questions about priorities or release timelines. If you work on a VMware product, I would also recommend including that information as well as your chain of command to raise up any concerns.

Thanks.

**Reply by wghglory** [reply]
Thank you @mathisscott ! Yes I work on VMware products. WIll reach out to the new team manager Steve. Thank you!
